flume/doc/design/OrionTestSink.md

```diff
@@ -0,0 +1,23 @@
+# OrionTestSink
```

Comment:
What about the existing sinks for HDFS, CKAN and MySQL? Have documentation being refactor to include their information as separate .md files as this one?